src/resolvers/utils.ts

```diff
@@ -74,7 +74,7 @@ export function convertToTagSaveAssociations(
  * @param attr The attribute of the parent to request
  * @returns The value keyed by `attr` of the parent or fetched object.
  */
-export async function lazyParentLoad<T, K, A extends keyof T>(
+export async function lazyParentLoad<T extends object, K, A extends keyof T>(
```

Comment:
I did this change based on some quick internet research. Looks like the latest typescript required this.